lib/gc.cpp

```diff
@@ -14,6 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// This include is necessary to prevent this error from happening:
+// "error: 'posix_memalign' is missing exception specification 'noexcept'"
+// This error is possible when compiling with clang and including "mm_malloc.h" after the definition
+// of posix_memalign. This can easily happen in unity builds.
+#if defined(__clang__)
+#include <mm_malloc.h>
```

Comment:
Is this header clang-specific?

Reply:
Actually, this is what I am using to fix build failures (https://github.com/p4lang/p4c/pull/4595). Shouldn't be in this PR.